Notes-CaaSP.md

```diff
@@ -132,11 +132,177 @@ kubectl create secret generic ceph-secret-admin
 
-### Provide enough disk space for images
 
-The default is to provide 40 GB of disk space to the CaaSP nodes. This is not sufficient for CAP. So use a machine flavor with a bigger disk and let is use the additional free space for storing images.
+### Considerations prior to deploying
 
-Use the free space, create a partition on it, format it with btrfs and mount it as
-`/var/lib/docker` before you start importing any containers. If you resize the root filesystem to the full disk, you have very big root filesystem and still no space to store the containers without a big performance penalty.
+* Provide enough disk space for images
+
+  The default is to provide 40 GB of disk space to the CaaSP nodes. This is not sufficient for CAP. So use a machine flavor with a bigger disk and let it use the additional free space for storing images (Commands on how to resize the CaaSP node root fs are provided in the instructions below).
+
+
+### Initial preparations
+
+The following steps only have to be done once before the initial CaaSP deployment. For following deplyments of CaaSP this has no to be redone but already created elements can be reused. The deployment of CaaSP on OpenStack is done using existing terraform rules with some additional steps required for running CAP on the CaaSP deployment.
+
+
+* Start with downloading and sourcing the openrc.sh file for OpenStack API access
+
+  ```
+  firefox https://$OPENSTACK/project/access_and_security/api_access/openrc/
+  . openrc.sh
+  ```
+
+  *Note* You will have to login to download the file. The filename might have a prefix named after the OpenStack project the file is for.
+
+
+**Optional steps**
+
+These step can be performed but are not mandatory. You can also use already existing OpenStack objects instead (e.g. if you do not have the permission to create projects or networks).
+
+* Create a openstack project to run CaaSP in (e.g. caasp), add a user as admin and export the project to be used by terraform
+
+  ```
+  openstack project create --domain default --description "CaaSP Project" caasp
+  openstack role add --project caasp --user admin admin
+  export OS_PROJECT_NAME='caasp'
+  ```
+
+* Create a Openstack network plus a subnet for caasp (e.g. caasp-net) and add a router to the extrenal (e.g. floating) network
+
+  ```
+  openstack network create caasp-net
+  openstack subnet create caasp_subnet --network caasp-net --subnet-range 10.0.2.0/24
+  openstack router create caasp-net-router
+  openstack router set caasp-net-router --external-gateway floating
+  openstack router add subnet caasp-net-router caasp_subnet
+  ```
+
+**Mandatory Steps**
+
+The following steps have to be done at least once in order to be able to deploy CaaSP and CAP on top of OpenStack
+
+
+* Clone the terraform script
+
+  ```
+  git clone git@github.com:kubic-project/automation.git
+  cd automation/caasp-openstack-terraform
+  ```
```

Comment:
I would suggest to move this section to after creating the security groups. Then the terraform steps are all together.